<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_setup_game</class>
 <widget class="QDialog" name="Dialog_setup_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>364</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>682</width>
    <height>364</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>682</width>
    <height>364</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game Setup</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Setup options for starting a new game.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>9</y>
     <width>666</width>
     <height>349</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_main">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit_campaign">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_challenge">
      <property name="text">
       <string>Single Challenge</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="4">
     <widget class="Line" name="hline_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <widget class="Line" name="hline_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_ok_challenge">
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_custom">
      <property name="text">
       <string>Custom Game</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_campaign">
      <property name="text">
       <string>Campaign code</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSlider" name="horizontalSlider_challenge">
      <property name="maximum">
       <number>1999</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
      <property name="tickInterval">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_ok_campaign">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="pushButton_ok_custom">
      <property name="text">
       <string>Start Game</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <layout class="QGridLayout" name="gridLayout_custom">
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_gravity">
        <property name="maxCount">
         <number>2147483644</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_gravity">
        <property name="text">
         <string>Gravity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBox_energy">
        <property name="toolTip">
         <string extracomment="Mean system energy per height level counting trees only. 1 tree is the rough equivalent of 1e19J."/>
        </property>
        <property name="suffix">
         <string>e17 J/m²</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_energy">
        <property name="text">
         <string>System energy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_age">
        <property name="text">
         <string>Geol. age</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_age"/>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox_sentries">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="specialValueText">
         <string>Randomized</string>
        </property>
        <property name="minimum">
         <number>-1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_sentries">
        <property name="text">
         <string>Sentries</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="3">
     <layout class="QGridLayout" name="gridLayout_dim">
      <item row="3" column="0">
       <widget class="QLabel" name="label_fov">
        <property name="text">
         <string>Antagonist FOV</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBox_confidence">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_spin_period">
        <property name="text">
         <string>Spin period</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_self_spin">
        <property name="text">
         <string>Self spin</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBox_psi_shield">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_rotation_type">
        <property name="text">
         <string>Rotation type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_rotation_type"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_psi_shield">
        <property name="text">
         <string>Psi shield</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_confidence">
        <property name="text">
         <string>Confidence</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_self_spin">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>24</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_spin_period">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>60</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBox_antagonist_fov">
        <property name="suffix">
         <string>°</string>
        </property>
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkBox_random_scenery">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Rnd scenery</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_object_resilience">
        <property name="text">
         <string>Object resilience</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCheckBox" name="checkBox_meanies">
        <property name="text">
         <string>Meanies</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBox_object_resilience">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="spinBox_cols">
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_cols">
        <property name="text">
         <string>Board columns</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBox_rows">
        <property name="minimum">
         <number>20</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_rows">
        <property name="text">
         <string>Board rows</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="4">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_restore_defaults">
        <property name="text">
         <string>Restore Defaults</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_general">
      <property name="text">
       <string>General Settings</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="4">
     <widget class="Line" name="hline_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_campaign</tabstop>
  <tabstop>pushButton_ok_campaign</tabstop>
  <tabstop>horizontalSlider_challenge</tabstop>
  <tabstop>pushButton_ok_challenge</tabstop>
  <tabstop>comboBox_gravity</tabstop>
  <tabstop>spinBox_sentries</tabstop>
  <tabstop>comboBox_age</tabstop>
  <tabstop>spinBox_energy</tabstop>
  <tabstop>pushButton_ok_custom</tabstop>
  <tabstop>comboBox_rotation_type</tabstop>
  <tabstop>spinBox_psi_shield</tabstop>
  <tabstop>spinBox_spin_period</tabstop>
  <tabstop>spinBox_confidence</tabstop>
  <tabstop>spinBox_self_spin</tabstop>
  <tabstop>spinBox_object_resilience</tabstop>
  <tabstop>spinBox_antagonist_fov</tabstop>
  <tabstop>checkBox_random_scenery</tabstop>
  <tabstop>checkBox_meanies</tabstop>
  <tabstop>spinBox_rows</tabstop>
  <tabstop>spinBox_cols</tabstop>
  <tabstop>pushButton_restore_defaults</tabstop>
  <tabstop>pushButton_close</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
